kb.py

```
@@ -0,0 +1,584 @@
+import requests
+import json
+import urllib.parse
+import time
+import os
+import subprocess
```

Ruff (0.12.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)